openforcefield/utils/toolkits.py

```diff
@@ -1040,6 +1041,80 @@ def to_smiles(molecule):
             | oechem.OESMILESFlag_AtomStereo)
         return smiles
 
+    @staticmethod
+    def to_inchi(molecule, fixed_hydrogens=False):
+        """
+        Create an InChI string for the molecule using the RDKit Toolkit.
+        InChI is a standardised representation that does not capture tautomers unless specified using the fixed hydrogen
+        layer.
+
+        For information on InChi see here https://iupac.org/who-we-are/divisions/division-details/inchi/
+
+        Parameters
+        ----------
+        molecule : An openforcefield.topology.Molecule
+            The molecule to convert into a SMILES.
+
+        fixed_hydrogens: bool, default=False
+            If a fixed hydrogen layer should be added to the InChI, if `True` this will produce a non standard specific
+            InChI string of the molecule.
+
+        Returns
+        --------
+        inchi: str
+            The InChI string of the molecule.
+        """
+
+        from openeye import oechem
+        oemol = OpenEyeToolkitWrapper.to_openeye(molecule)
```

Comment:
Per the discussion on "what should be a class method vs. instance method" last week, I think having to write something like this means that this should be an instance method and that this line should be `oemol = self.to_openeye(molecule)`.

I know it's used inconsistently across the toolkits, but starting the transformation to instancemethods now can let us make CachingToolkitWrappers a drop-in replacement later.

Reply:
Yeah I think you're right this is quite awkward to write this way, Ill change these over.